src/peergos/shared/crypto/password/PasswordProtected.java

```diff
@@ -13,7 +13,7 @@
 public class PasswordProtected {
 
     public static SecretGenerationAlgorithm getDefault() {
-        return new ScryptGenerator(ScryptGenerator.MIN_MEMORY_COST, 8, 1, 32);
+        return new ScryptGenerator(ScryptGenerator.MIN_MEMORY_COST, 8, 1, 32, "");
```

Comment:
To be clear, we consider "no salt" to be the default behaviour?

Reply:
Current users have no extra salt, just the username as salt. New users after this pr, and existing users once they change their password will use a 32 random byte (converted to hex) salt.

Reply:
Got it 👍